amf-client/shared/src/main/scala/amf/client/commands/CommandHelper.scala

```diff
@@ -50,8 +50,7 @@ trait CommandHelper {
       Option(effectiveMediaType(config.inputMediaType, config.inputFormat)),
       config.inputFormat,
       Context(platform),
-      cache = Cache(),
-      errorHandler = UnhandledParserErrorHandler
```

Comment:
This change is strange, do we know the real reason why this is failing? Beside the symptom.

Reply:
Yes, the short version is that the `UnhandledParserErrorHandler ` only throws the validations, and omits adding them to the `StaticErrorCollector`. 

Inside `UnhandledParserErrorHandler` you can see that it just throws an exception and nothing more:
```scala
override def handlerAmfResult(result: AMFValidationResult): Boolean = {
    throw new Exception(
      s"  Message: ${result.message}\n  Target: ${result.targetNode}\nProperty: ${result.targetProperty.getOrElse("")}\n  Position: ${result.position}\n at location: ${result.location}")
  }
```

Reply:
Right but why doesn't the CLI's execution stop on an Exception?

Reply:
that's something to analyze. I think it keeps validating and then returns all warnings/violations. But none are returned because none are stored.